src/ObjectUtils/TestLibrary.cpp

```diff
@@ -7,4 +7,33 @@
 
 #include <cstdio>
 
```

Comment:
I just realized: In the internal build this file is compiled into the ObjectUtils module. Could you do me a favor and either move the file into testdata/ (probably preferred) or add it the exclude list in the orbit_cc_library target in the `BUILD` file?